zeppelin-interpreter/src/main/java/org/apache/zeppelin/interpreter/InterpreterContext.java

```diff
@@ -124,4 +124,28 @@ public ResourcePool getResourcePool() {
     return runners;
   }
 
+  public String getRequiredReplName() {
```

Comment:
I forgot about this, this definitely sounds like it would be a problem. I was struggling to figure out ways to automatically detect the current interpreter class through `ZeppelinContext` since I thought that would be a rather convenient for users, but I couldn't come up with better ideas at the time. 

I think this problem can be partially addressed by by passing in the interpreter class name through the current `InterpreterContext` instance as that could solve the problem for automatically detecting the current interpreter that's in use. However this still wouldn't allow users to manually specify aliases for another interpreter that isn't keyed into the internal mapping. Let me know what you think of this solution and if you have any other ideas.